prime-router/src/testIntegration/resources/datatests/translation-test-config.csv

```diff
@@ -1,29 +1,33 @@
+HL7_to_FHIR_to_HL7/elims_29_5065302_35227_NoPII_CANCELED.hl7,,HL7_to_FHIR_to_HL7/elims_29_5065302_35227_NoPII_CANCELED-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","test.dev-elims","","","",""
+HL7_to_FHIR_to_HL7/hci.hl7,,HL7_to_FHIR_to_HL7/hci-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","test.dev-hci","","","",""
+FHIR_to_HL7/sample_SR_1_20230302-0001.fhir,,FHIR_to_HL7/sample_SR_1_20230302-0001.hl7,,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform",ca-test.default-receiver,"",""
+FHIR_to_HL7/sample_SR_1_20230302-0001-missing-data.fhir,,FHIR_to_HL7/sample_SR_1_20230302-0001-missing-data.hl7,,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform",ca-test.default-receiver,"",""
+FHIR_to_HL7/sample_SR_2_20231024-0002.fhir,,FHIR_to_HL7/sample_SR_2_20231024-0002.hl7,metadata/hl7_mapping/receivers/STLTs/NY/NY-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform","","",""
+FHIR_to_HL7/sample_SR_2_20231106-0003.fhir,,FHIR_to_HL7/sample_SR_2_20231106-0003.hl7,metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform","","",""
+FHIR_to_HL7/sample_OML_20230831-0001.fhir,,FHIR_to_HL7/sample_OML_20230831-0001.hl7,metadata/hl7_mapping/receivers/Flexion/TILabOrder,HL7,PASS,"","","","","",""
+FHIR_to_HL7/sample_SR_2_20231106-0003.fhir,,FHIR_to_HL7/sample_RI_20231213-0001.hl7,metadata/hl7_mapping/receivers/STLTs/RI/RI-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform","","",""
+FHIR_to_HL7/simple_report_with_date_aoe_question.fhir,,FHIR_to_HL7/simple_report_with_date_aoe_question.hl7,metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform","","",""
+HL7_to_FHIR/sample_orm_20230809-001.hl7,,HL7_to_FHIR/sample_orm_20230809-001-with-enrichment.fhir,,FHIR,PASS,"","","","","","/metadata/enrichment/testing/testing2"
+HL7_to_FHIR_to_HL7/elims_40_4988249_33033.hl7,,HL7_to_FHIR_to_HL7/elims_40_4988249_33033_with_enrichment.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-test,HL7,PASS,"","test.dev-elims","","","","/metadata/enrichment/testing/testing2"
+HL7_to_FHIR_to_HL7/elims_40_4988249_33033.hl7,,HL7_to_FHIR_to_HL7/elims_40_4988249_33033_with_double_enrichment.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-test,HL7,PASS,"","test.dev-elims","","","","/metadata/enrichment/testing/testing,/metadata/enrichment/testing/testing2"
+HL7_to_FHIR_to_HL7/elims_40_4988249_33033.hl7,,HL7_to_FHIR_to_HL7/elims_40_4988249_33033_with_double_enrichment.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-test,HL7,PASS,"","test.dev-elims","","ca-test.enrichment-receiver","",""
```

Comment:
I like this test and struggle to see the value of the other ones? I guess the one above this allows you to specify the enrichment schema name outside of the configured receiver, but even then, not sure we need that? Still not sure about the first two you added though. Seems to me if we updated this receiver to contains two enrichment schemas that would be all we need to test your new functionality?

Reply:
One checks it works on HL7 to FHIR (should be no difference, but figure it is still best to have a test in case someone mucks it up). One tests HL7 to HL7, but with only one schema name to make sure we don't get dependent on it being more than one name. One tests multiple schema names. One tests that we can use the configuration in the yml file as you asked me to do last week.

Reply:
The following two can be combined by specifying multiple in the receiver config, no?
- `One tests that we can use the configuration in the yml file`
- `One tests multiple schema names.`

Reply:
Sort of. It would mean that we aren't testing that the tests can handle both methods, but if you are fine with that, I can remove it.